src/BlueskyClient/Services/AuthenticationService.cs

```diff
@@ -87,7 +89,7 @@ public async Task<Result<AuthResponse>> SignInAsync(string rawUserHandleOrEmail,
             return Result.Fail<AuthResponse>("Empty identifier or password");
         }
 
-        return await SignInWithValidatedCredentialsAsync(userHandleOrEmail, password);
+        return await SignInWithValidatedCredentialsAsync(userHandleOrEmail, password, baseUrl ?? "bsky.social");
```

Comment:
After making my recommended changes on `BlueskyApiClient.cs`, then we don't need this null coalesce anymore. In fact, this is one of the reasons why the PR had an http error

```suggestion
        return await SignInWithValidatedCredentialsAsync(userHandleOrEmail, password, baseUrl);
```